scripts/trace-acceptor.sh

```diff
@@ -9,6 +11,7 @@ ${CMD} \
     --topology "./configuration/simple-topology.json" \
     --database-path "./db/" \
     --genesis-file "configuration/mainnet-genesis.json" \
+    --genesis-hash "5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb" \
```

Comment:
Shouldn't the genesis hash be pulled from `lib-node.sh`?

Reply:
@Jimbo4350, it's hard-coded to mainnet here, so no it can't be pulled from there.